### DISCUSSION TITLE
Improve Thumbnail Quality in Ente

### DISCUSSION THREAD

**Discussion by Jishnuraj9** [ente-io/ente#3075] [Enhancements]
The thumbnail quality in Ente isn't as good as it is on other platforms like Apple Photos.

Here are some screenshots comparing the thumbnail quality between Ente photos and Apple photos

### On Apple photos : 
![3](https://github.com/user-attachments/assets/6311b880-2e70-4bc4-bf4a-59f6956d0926)

### On Ente photos: 
![4](https://github.com/user-attachments/assets/69c4fa5a-c75d-4656-abbf-23209487f145)

It would be great if the thumbnail quality on Ente photos could be improved to match that of competing platforms.

**Comment by woodedjurassicpark** [comment]
Hi, were your example photos uploaded from the web? 

My thumbnails look like the above ones when uploaded from the iOS app. However, if I upload them from Ente Web using any browser they look horrendous. Some examples that really stand out are pictures with lined paper, panoramas, or particularly pictures of computer screens with moire patterns. The former looks really jagged and in the latter the moire pattern turns into a black mask that covers parts of the image in the thumbnails.

Also, if I upload from Ente Photos flatpak on Linux, they look blurry like the ones you have exampled here. Also of note, thumbnails only look good on Firefox based browsers or mobile apps.

I'm archiving(uploading) a bunch of older photos and this is problematic, because I have to upload them to my phone first. This is doubly problematic, because it renames files when automatically backed up through the iOS app, even though the filenames show correct on the phone.
